_posts/2016-11-28-course-collaboration.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+subheadline: "Curriculum"
+title: "Data Carpentry Expanding Collaborative Lesson Development to Semester-long Courses"
+teaser: "Let's build on Data Carpentry's core strength of collaborative lesson development."
+header:
```

Comment:
Just suggesting a slight title revision.
title: "Collaborative Lesson Development to Semester-long Courses"
teaser: "Building on Data Carpentry's core strength of collaborative lesson development."